### PR TITLE
ZW - AN - 6pm-3 - Refactor Course Table

### PR DESCRIPTION
**Pull request by Zwang1680**
Refactored the coursetable to include just the default arrows not our own premade ones. Also edited the test file to no longer check for the made arrows.

### CODE REVIEW

**Review by m-cabello — DISMISSED** [review]
Looks Good! You guys follow the instructions accurately/\.

File: javascript/src/test/components/Courses/CourseTable.test.js [hunk 1]
```diff
@@ -29,15 +29,19 @@ describe("CourseForm tests", () => {
...
     const { getByText } = render(<CourseTable courses={courses} admin={true} deleteCourse={deleteCourse} />);
     const courseNumberHeader = getByText(/Course Number/);
 
     //cycle through both ascending and descending for test coverage
     userEvent.click(courseNumberHeader);
     userEvent.click(courseNumberHeader);
-    const descendingOFF = String.fromCharCode(0x25bd);
-    expect(getByText(descendingOFF)).toBeInTheDocument();
+    // const descendingOFF = String.fromCharCode(0x25bd);
+    // expect(getByText(descendingOFF)).toBeInTheDocument();
```
Comment (WadeVaresio) [inline javascript/src/test/components/Courses/CourseTable.test.js:44]: Please remove these commented out lines, we typically do not want to include commented out code.

**Review by WadeVaresio — CHANGES_REQUESTED** [review]
LGTM, will be ready to merge once the commented lines are removed.

**Review by m-cabello — DISMISSED** [review]
Look Great!

**Review by JacquelineMai — APPROVED** [review]
LGTM
We've agreed to let this merge with the codecov failures, since we've confirmed that the test coverage isn't actually decreasing.

**Review by mpbonham — APPROVED** [review]
Everything looks good!

**Pull request merged**